front/lib/api/workspace.ts

```diff
@@ -518,3 +527,81 @@ export async function upgradeWorkspaceToBusinessPlan(
 
   return new Ok(undefined);
 }
+
+export async function checkSeatCountForWorkspace(
+  workspace: LightWorkspaceType,
+  updateQuantity: boolean
```

Comment:
Let's avoid the native types outside of objects.


```suggestion
  {updateQuantity}: {updateQuantity: boolean}
```